tools/release/adot-operator-images-mirror/config.yaml

```diff
@@ -19,9 +19,47 @@ sourceRepos:
       - v0.11.0
       - v0.8.0
       - v0.5.0
+  - registry: open-telemetry/opentelemetry-operator
+    name: target-allocator
+    host: ghcr.io
+    allowed_tags:
+      - 0.1.0
+  - registry: open-telemetry/opentelemetry-operator
+    name: autoinstrumentation-java
+    host: ghcr.io
+    allowed_tags:
+      - 1.11.1
```

Comment:
Double check the list of required tags.  There are [several newer tags](https://github.com/open-telemetry/opentelemetry-operator/pkgs/container/opentelemetry-operator%2Fautoinstrumentation-java/versions?filters%5Bversion_type%5D=tagged) in the repo that probably should also be mirrored.  The same goes for, at least, the dotnet images as well.